db/migrate/20250603164430_update_tenant_name.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateTenantName < ActiveRecord::Migration[7.0]
+  class SettingsChange < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  class Tenant < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  def change
+    # there may be multiple of these, but it should only be
+    sc = SettingsChange.in_my_region.find_by(:key => "/server/company")
+    Tenant.find_by(:ancestry => nil, :use_config_for_attributes => true)&.update(:name => sc&.value || "My Company", :use_config_for_attributes => false)
+    SettingsChange.in_my_region.where(:key => "/server/company").destroy_all
+  end
+
+  def revert
+    # no real way to revert
+    # but since the tenant record no longer looks into server/company record, it is fine
```

Comment:
In a revert situation we'd be using old code, so it _would be loking at the settings.  I think this comment should be removed.